doc/theory/glossary.tex

```diff
@@ -339,30 +494,54 @@
 	
 	Let $D = \{(b_i, d_i)\}_{i \in I}$ be a
 	%\hyperref[persistence_diagram]{persistence diagram}
-	persistence diagram and $w = \{w_i\}_{i \in I}$ a set of positive real numbers. The \textit{silhouette of $D$ weighted by $w$} is the function $\phi : \mathbb R \to \mathbb R$ defined by	
+	persistence diagram and $w = \{w_i\}_{i \in I}$ a set of positive real numbers. The \textit{silhouette of} $D$ \textit{weighted by} $w$ is the function $\phi : \mathbb R \to \mathbb R$ defined by
 	\begin{equation*}	
 	\phi(t) = \frac{\sum_{i \in I}w_i \Lambda_i(t)}{\sum_{i \in I}w_i},	
 	\end{equation*}	
 	where	
 	\begin{equation*}
 	\Lambda_i(t) = \left[ \min \{t-b_i, d_i-t\}\right]_+	
 	\end{equation*}	
-	and $c_+ := \max(c,0)$. When $w_i = \vert d_i - b_i \vert^p$ for $0 < p \leq \infty$ we refer to $\phi$ as the \textit{$p$-power-weighted silhouette} of $D$. The silhouette construction defines a
-	%\hyperref[vectorization_kernel_and_amplitude]{vectorization}
-	vectorization of the set of persistence diagrams with target the vector space of continuous real-valued functions on $\mathbb R$.	
+	and $c_+ := \max(c,0)$. When $w_i = \vert d_i - b_i \vert^p$ for $0 < p \leq \infty$ we refer to $\phi$ as the $p$-\textit{power-weighted silhouette} of $D$. The silhouette construction defines a
+	%\hyperref[vectorization_amplitude_and_kernel]{vectorization}
+	vectorization of the set of persistence diagrams with target the vector space of continuous real-valued functions on $\mathbb R$.
 	
 	\paragraph{\\ References:} \cite{chazal2014stochastic}
 	
+	\subsection*{Heat vectorizations} \label{heat_vectorization}
+	
+	Considering the points in a persistence diagram as the support of Dirac deltas one can construct, for any $t > 0$,
+	%\hyperref[vectorization_amplitude_and_kernel]{vectorization}
+	two vectorizations of the set of persistence diagrams to the set of continuous real-valued function on the first quadrant $\mathbb{R}^2_{>0}$. The \textit{symmetry heat vectorization} is constructed for every persistence diagram $D$ by solving the heat equation
+	\begin{align} \label{equation: heat equation}
+	\Delta_x(u) &= \partial_t u && \text{on } \Omega \times \mathbb R_{>0} \nonumber \\
+	u &= 0 && \text{on } \{x_1 = x_2\} \times \mathbb R_{\geq 0} \\
+	u &= \sum_{p \in D} \delta_p && \text{on } \Omega \times {0} \nonumber
+	\end{align}
+	where $\Omega = \{(x_1, x_2) \in \mathbb R^2\ |\ x_1 \leq x_2\}$, then solving the same equation after precomposing the data of \eqref{equation: heat equation} with the change of coordinates $(x_1, x_2) \mapsto (x_2, x_1)$, and defining the image of $D$ to be the difference between these two solutions at the chosen time $t$.
+	
+	Similarly, the \textit{rotation heat vectorization} is defined by sending $D$ to the solution, evaluated at time $t$, of the equation obtained by precomposing the data of \eqref{equation: heat equation} with the change of coordinates $(x_1, x_2) \mapsto (x_1, x_2-x_1)$.
```

Comment:
I note that there is a tension here between using "Persistence image" in the library, and "rotation heat vectorization" here. Not asking for a change here as it is beyond the scope for this PR.